webapp/locales/pt.json

```diff
@@ -321,7 +321,63 @@
             "disabledBy": "desativado por",
             "reasonCategory": "Categoria",
             "reasonDescription": "Descrição",
-            "createdAt": "Data"
+            "status": "",
```

Comment:
@Mogge we must make sure that we don't add empty strings to our locales. Either leave it out or add `"status": null` to the translation files. Otherwise we disable the English fallback and the user will see a blank page.

Reply:
I check this. I tried to reconfigure the @appinteractive , just to match the new semantic requirements but failed so far